numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -2406,6 +2422,36 @@ def get_function(self, name):
     def get_global_symbol(self, name):
         """Return a MemoryPointer referring to the named symbol"""
 
+    def setup(self):
+        """Call the setup functions for the module"""
+        if self.initialized:
+            raise RuntimeError("The module has already been initialized.")
+
+        if self.setup_functions is None:
+            return
+
+        for f in self.setup_functions:
+            f(self.handle)
+
+        self.initialized = True
+
+    def _set_finalizers(self):
+        """Create finalizers that tears down the module.
```

Comment:
A tiny nit (which you might want to resolve if you're fixing things up for the `LinkableCode` docstring below):
```suggestion
        """Create finalizers that tear down the module.
```